docs/CONFIG-VARS.md

```diff
@@ -54,22 +54,26 @@ For recommendations on how to set these variables in your environment, see [Auth
 
 ## Admin Access
 
-By default, the API of the Azure resources that are being created is only accessible through authenticated Azure clients (such as the Azure Portal,
-the `az` CLI, the Azure Shell, etc.).
-To allow access for other administrative client applications (for example `kubectl`, `psql`, etc.), you must open the Azure firewall to allow access from your source
-IP addresses.
-To do this, specify ranges of IP addresses in [CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing).
-Contact your Network Administrator to find the public CIDR range of your network.
+By default, the public endpoints of the Azure resources that are being created
+are only accessible through authenticated Azure clients
+(such as the Azure Portal, the `az` CLI, the Azure Shell, etc.).
+To allow access for other administrative client applications (for example `kubectl`, `psql`, `ssh` etc.), you can set Network Security Group (NSG) rules to control access from your source IP addresses.
```

Comment:
So there is no longer an option for Azure Firewall as previously stated. This has been removed and you must only use a NSG?

Reply:
The previous version did use the term "firewall", but it was used as a generic term, not as a reference to the "Azure Firewall" object. I want to argue that being more specific about how we do the access control, we avoid possible misunderstandings, and make it clearer how the _access_cidr variables are used.